node/src/actors/session/mod.rs

```diff
@@ -129,34 +126,63 @@ impl Session {
         // Convert WitnetMessage into a vector of bytes
         match ProtobufConvert::to_pb_bytes(&msg) {
             Ok(bytes) => {
-                debug!(
-                    "{} Sending  {} message to session {:?} ({} bytes)",
-                    Green.bold().paint("[>]"),
-                    Green.bold().paint(msg.kind.to_string()),
-                    self.remote_addr,
-                    bytes.len(),
-                );
-                trace!("\t{:?}", msg);
+                match msg.kind {
+                    Command::Transaction(_) | Command::Block(_) => {
+                        let log_data = format!(
+                            "{} Sending {} message ({} bytes)",
+                            Green.bold().paint("[>]"),
+                            Green.bold().paint(msg.kind.to_string()),
+                            bytes.len(),
+                        );
+                        SessionsManager::from_registry().do_send(LogMessage { log_data })
+                    }
+                    _ => {
+                        log::debug!(
+                            "{} Sending  {} message to session {:?} ({} bytes)",
+                            Green.bold().paint("[>]"),
+                            Green.bold().paint(msg.kind.to_string()),
+                            self.remote_addr,
+                            bytes.len(),
+                        );
+                    }
+                }
+                log::trace!("\t{:?}", msg);
                 self.framed.write(bytes.into());
             }
             Err(e) => {
-                error!(
+                log::error!(
                     "Error sending {} message to session {:?}: {}",
-                    msg.kind, self.remote_addr, e,
+                    msg.kind,
+                    self.remote_addr,
+                    e,
                 );
-                trace!("\t{:?}", msg);
+                log::trace!("\t{:?}", msg);
             }
         }
     }
     // This method is useful to align the logs from receive_message with logs from send_message
     fn log_received_message(&self, msg: &WitnetMessage, bytes: &[u8]) {
-        debug!(
-            "{} Received {} message from session {:?} ({} bytes)",
-            Green.bold().paint("[<]"),
-            Green.bold().paint(msg.kind.to_string()),
-            self.remote_addr,
-            bytes.len(),
-        );
-        trace!("\t{:?}", msg);
+        match msg.kind {
+            Command::Transaction(_) | Command::Block(_) => {
+                let log_data = format!(
+                    "{} Received {} message ({} bytes)",
+                    Green.bold().paint("[<]"),
+                    Green.bold().paint(msg.kind.to_string()),
+                    bytes.len(),
+                );
+                SessionsManager::from_registry().do_send(LogMessage { log_data })
```

Comment:
Maybe we can also save `self.remote_addr`, even if we do not print it, so that we don't lose that information. This would mean changing the `HashSet<String>` into a `HashMap<String, Vec<SocketAddr>>`.

Reply:
This PR is only to change the way that we show the messages, but it doesnt affect to the witnet logic, so... why we would need to save that information?